frontend/public/components/factory/list-page.jsx

```diff
@@ -242,22 +246,24 @@ export const FireMan_ = connect(null, { filterList })(
       return (
         <>
           {/* Badge rendered from PageHeading only when title is present */}
-          <PageHeading
-            title={title}
-            badge={title ? badge : null}
-            className={classNames({ 'co-m-nav-title--row': createLink })}
-          >
-            {createLink && (
-              <div
-                className={classNames('co-m-pane__createLink', {
-                  'co-m-pane__createLink--no-title': !title,
-                })}
-              >
-                {createLink}
-              </div>
-            )}
-            {!title && badge && <div>{badge}</div>}
-          </PageHeading>
+          {!hidePageHeading && (
```

Comment:
Is `showTitle` not able to handle the issue?

Reply:
It turns out that `showTitle` is used in dozens of places and I am removing the entire page heading (the title,  the padding, and the create button) which is a big change if people are not expecting it.  I therefore decided to use a new flag, which I know adds even more special flags but it seems like the most backwards-compatible solution.
I think we might want to consider making a new variable throughout the list page ecosystem that basically holds an array of these flags so we do not need to keep adding all this in the future, but that would be feature work for the future (like an `options` variable).

Reply:
> It turns out that `showTitle` is used in dozens of places and I am removing the entire page heading (the title, the padding, and the create button) which is a big change if people are not expecting it.

We already have props for hiding the title and the create button. If the only thing we need is padding, that seems like it could be handled with a CSS class.